### PR TITLE
Add flac and libsndfile.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] REQUIRED - The package is added to `packages.txt`

#### For security-related PRs

- [ ] The security fix is recorded in `advisories` and `secfixes`

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

**Comment by kaniini** [comment]
Looks like `speex` is not in wolfi yet, but `speexdsp` is.

File: flac.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      - glibc-iconv
+      - samurai
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 6c58e69cd22348f441b861092b825e591d0b822e106de6eb0ee4d05d27205b70
+      uri: https://downloads.xiph.org/releases/flac/flac-${{package.version}}.tar.xz
+
+  - runs: |
```
Comment (kaniini) [inline flac.yaml:29]: Why not use `cmake/*` pipelines here?  Then you could drop the `samurai` and `cmake` deps.

**Comment by dlorenc** [comment]
> Looks like `speex` is not in wolfi yet, but `speexdsp` is.

Yeah typo

File: libsndfile.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      - automake
+      - autoconf
+      - opus-dev
+      - opus
+      - alsa-lib-dev
+      - cmake
+      - flac-dev
+      - libogg-dev
+      - libvorbis-dev
+      - speexdsp-dev
```
Comment (kaniini) [inline libsndfile.yaml:25]: libspeex and libspeexdsp are different, one is for loading speex files (which I know libsndfile has a backend for), while the other is for voice chat mostly.

**Review by kaniini — DISMISSED** [review]

**Pull request merged**